ethexe/cli/src/tests.rs

```diff
@@ -692,9 +766,13 @@ async fn ping_deep_sync() {
     log::info!("📗 Waiting for code loaded");
     listener
         .apply_until(|event| {
-            if let Event::CodeLoaded(loaded) = event {
-                assert_eq!(loaded.code_id, code_id);
-                assert_eq!(loaded.code.as_slice(), demo_ping::WASM_BINARY);
+            if let Event::CodeLoaded {
+                code_id: loaded_id,
+                code,
+            } = event
+            {
```

Comment:
What the reason of this lines amount increase anywhere in tests ?

Reply:
rust formatting, replaced if let with matching - become a little shorter

Reply:
In separate pr we can impl some handlers like:
`event.on_code_loaded(|id, code| /* do smith */)`